@ndlib/gatsby-theme-marble/src/components/Shared/Seo/SeoContent/index.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { GatsbySeo } from 'gatsby-plugin-next-seo'
+import WebsiteJsonLd from './WebsiteJsonLd'
```

Comment:
Should this be on every page or just the index and/or search pages?

Reply:
No. This should only be on the homepage.

Reply:
Unless I'm misreading this, it appears to get placed on every page.So this should likely on in marble-web on the index page only.